map-card.js

```diff
@@ -922,7 +931,13 @@ class HaMapUtilities {
         return date;  
       }  
     }  
-  }  
+  }
+
+  // Show error message
+  static renderWarningOnMap(map, message){
```

Comment:
in this case would be good to handle the reset logic as part of the same class as well.

Reply:
Have updated the warnings to all use Util version + added the remove warning variant too. 
I also tweaked prefix while i was at it to differentiate warnings a little more
![image](https://github.com/nathan-gs/ha-map-card/assets/887397/9722161e-fd2f-4384-946f-c04060b0bbf7)